package.json

```diff
@@ -77,12 +77,14 @@
     "react-dnd": "^14.0.2",
     "react-dom": "^16.8.1",
     "react-ellipsis-with-tooltip": "^1.0.8",
+    "react-bootstrap": "^0.32.0",
     "react-helmet": "^6.1.0",
     "react-intl": "^2.8.0",
     "react-loading-skeleton": "^1.1.2",
     "react-onclickoutside": "^6.6.2",
     "react-password-strength": "^2.4.0",
     "react-redux": "^7.1.0",
+    "react-router": "^5.3.4",
```

Comment:
Maybe also in config/webpack.vendor.js

Reply:
It builds without, but I have no idea if that's correct :)